packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2754,13 +2754,15 @@ function updateDehydratedSuspenseComponent(
   }
 }
 
-function scheduleWorkOnFiber(fiber: Fiber, renderLanes: Lanes) {
+function scheduleSuspenseWorkOnFiber(fiber: Fiber, renderLanes: Lanes) {
   fiber.lanes = mergeLanes(fiber.lanes, renderLanes);
   const alternate = fiber.alternate;
   if (alternate !== null) {
     alternate.lanes = mergeLanes(alternate.lanes, renderLanes);
   }
-  scheduleWorkOnParentPath(fiber.return, renderLanes);
+  // Guaranteed to be non-empty since the fiber is not a root.
+  const parentFiber: Fiber = (fiber.return: any);
+  scheduleContextWorkOnParentPath(parentFiber, renderLanes, parentFiber);
```

Comment:
I've started passing the parent fiber here. So for Suspense changes, its parent is set as a propagation root. Is that reasonable?